src/components/interfaces/MOBILE_API.xml

```diff
@@ -3370,6 +3370,27 @@
         <element name="HAZARD_LIGHTS" value="511">
             <description>Include all hazard lights: front_left, front_right, rear_left and rear_right.</description>
         </element>
+        <element name="REAR_CARGO_LIGHTS" value="512" since="5.0">
```

Comment:
everything in this enum is since 5.0, so it does not need this in each new element

Reply:
@yang1070 please take a look 8f010d6328f620c8765130fdaa66e4df0fcf33fb